get_dem.py

```diff
@@ -1,106 +1,179 @@
-import os
 import argparse
-import subprocess
-from osgeo import gdal
-import numpy as np
+import logging
+import os
+from dataclasses import dataclass
+from functools import wraps
 from time import time
 
+import numpy as np
+from osgeo import gdal  # type: ignore
+from sardem import cop_dem  # type: ignore
+
 __version__ = "0.2.0"
 
-def get_dem(bbox: str, out_dir: str) -> str:
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(name)s] [%(levelname)s] %(message)s",
+    datefmt="%Y-%m-%d %H:%M:%S",
+)
+
+# Remove the sardem logging handler so we can control the output, because the
+# cop_dem module adds its own handler (poor practice for a library to do).
+logging.getLogger("sardem").handlers.clear()
+logger = logging.getLogger("get_dem")
+
+
+@dataclass(frozen=True, kw_only=True)
```

Comment:
Just a comment --> `kw_only` [appeared with Python 3.10](https://docs.python.org/3/library/dataclasses.html#:~:text=in%20version%203.10.-,kw_only,-%3A%20If%20true), so I suppose this determines the minimum Python version for the `environment.yml`. Good thing that worked out!